Loop/Managers/DeviceDataManager.swift

```diff
@@ -416,7 +415,8 @@ final class DeviceDataManager {
             alertManager: alertManager,
             analyticsServicesManager: analyticsServicesManager,
             loggingServicesManager: loggingServicesManager,
-            remoteDataServicesManager: remoteDataServicesManager
+            remoteDataServicesManager: remoteDataServicesManager,
+            servicesManagerDelegate: self
```

Comment:
Dosing (eventually) needs to be delegated to DeviceDataManager, but overrides, carb, settings, are more the domain of LoopDataManager.  And we may want to even run dosing through LoopDataManager, as a bolus command received mid-loop is something to consider.